server/services/licenceService.ts

```diff
@@ -388,8 +388,8 @@ export default class LicenceService {
   ): Promise<void> {
     const requestBody = {
       username: user.username,
-      eventTime: moment(eventTime).format('DD/MM/YYYY hh:mm:ss'),
-      eventType: user ? 'USER_EVENT' : 'SYSTEM_EVENT',
+      eventTime: moment(eventTime).format('DD/MM/YYYY HH:mm:ss'),
+      eventType: user && user?.username !== 'SYSTEM' ? 'USER_EVENT' : 'SYSTEM_EVENT',
```

Comment:
When is the username 'SYSTEM'?

Reply:
I don't think this is ever the case?

Reply:
You're right. When called from the UI service we always pass the current `user` which will set the event type to USER_EVENT. All other auditing is done inside the API. I'll remove this extra check.

Reply:
I think it was the update status after a date change (now removed) that might have been why I added this - because we used to record the edited licence as a USER EVENT there.

Reply:
Pushed again now.